packages/@icon-magic/distribute/src/create-sprite.ts

```diff
@@ -27,6 +27,7 @@ export async function addToSprite(
   spriteNames: SpriteConfig
 ): Promise<void> {
   let DOCUMENT, svgEl;
+  const promises: any[] = [];
```

Comment:
Nitpick... more specific? `Promise<void>[]`? (Unless the promise resolves to something.)